js-exercises/cache-function/cacheFunction.js

```diff
@@ -0,0 +1,15 @@
+const cacheFunction = (fn) => {
+  let cache = {}
+  return (...args) => {
+    let n = args[0]
+    if (n in cache) {
```

Comment:
The `in` operator climbs the prototype chain as well. You should a different methods, such as `hasOwnProperty`